src/MCPClient/tests/test_create_dataverse_mets.py

```diff
@@ -211,3 +251,31 @@ def test_mets_content(self, fixture, tmpdir):
                 pytest.fail("Unexpected MDREF found in metadata: {}"
                             .format(ref.get("LABEL")))
         assert dataset_json is True
+
+    @pytest.mark.parametrize(
+        "fixture", [dv_1, dv_2, dv_3, dv_4, dv_5, dv_6, dv_7, dv_8])
+    def test_ddi_authors(self, fixture, tmpdir):
+        """Test that we have retrieved the correct number of authors and that
+        the list compares 1:1 in order.
+        """
+        mets_path = self._create_mets(fixture.fname, tmpdir)
+        if not os.path.isfile(mets_path):
+            pytest.fail("Fixtures were not previously setup correctly.")
+
+        try:
+            mets = metsrw.METSDocument.fromfile(mets_path)
+        except mets.MetsError:
+            pytest.fail("Could not parse mets {}".format(mets_path))
```

Comment:
`metsrw.MetsError` would be the exception error. This is happening in other tests in this same file.

There is this tension between simpler tests and friendlier error messages. You could just let it raise, the developer is going to be focusing on the traceback anyways.